### DISCUSSION TITLE
Error handling

### DISCUSSION THREAD

**Discussion by plompd** [Dotnet-Boxed/Templates#23] [Q&A]
I'm pretty new to MVC error handling. 
I created a MVC5 project using your template. I would expect that if I just entered a non-existing url that I would see a 404 page, but it gives me the default IIS page.

What should I do to 'enable' the whole error handling / pages infrastructure?

**Comment by RehanSaeed** [comment]
If you do a build in release mode, it will do that. There is a mode attribute on the httpErrors section in web.config which has to be changed in Debug mode to enable debug error pages with the full exception details. See Web.config and Web.Release.config for more details.

```
    <!-- existingResponse - Whether or not to enable custom error pages.
                            PassThrough - Allow the default MVC debug error page showing the full exception to display
                            Replace - Replace any error responses with custom error pages. -->
    <httpErrors errorMode="Custom" existingResponse="PassThrough">
```
